pkg/minikube/config/profile_test.go

```diff
@@ -277,7 +279,11 @@ func TestGetPrimaryControlPlane(t *testing.T) {
 			t.Fatalf("Failed to load config for %s", tc.description)
 		}
 
-		n, err := PrimaryControlPlane(cc)
+		if tc.description == "old style" {
+			viper.Set(ProfileName, tc.profile+"_converted")
```

Comment:
I don't think this is correct. The profile name should be `tc.profile`. If we need things to be different for test data, we need to move the test data.